projectDocs/dev/copyrightHeaders.md

```diff
@@ -0,0 +1,26 @@
+In NVDA files, which are licensed under the GNU GPL, use the following copyright header:
+
+```
+# A part of NonVisual Desktop Access (NVDA)
+# Copyright (C) <CREATED YEAR>-<LAST UPDATED YEAR> NV Access Limited, <YOUR NAME>
+# This file may be used under the terms of the GNU General Public License, version 2 or later.
+# For more details see: https://www.gnu.org/licenses/gpl-2.0.html
+```
+
+Replacing:
+- `<YOUR NAME>` with your name.
+- `<CREATED YEAR>` the year the file was created
+- `<LAST UPDATED YEAR>`when the file was last updated. For new files this can be missing. E.g. a file created in 2025 might have `#Copyright (C) 2025`
```

Comment:
```suggestion
- `<LAST UPDATED YEAR>` when the file was last updated.
For new files this can be missing.
e.g. a file created in 2025 might have `# Copyright (C) 2025`
```